config/example.yaml

```diff
@@ -0,0 +1,18 @@
+server:
```

Comment:
Good point. Adding middlewares like metrics (e.g. [strong-express-metrics](https://www.npmjs.com/package/strong-express-metrics), [express-node-metrics](https://www.npmjs.com/package/express-node-metrics) or [express-metrics](https://www.npmjs.com/package/express-metrics)) is pretty easy and straightforward. Since I am not pretty sure what middlewares we want to add to our APIs, I only added the healthcheck middleware to the `/staff-fee-privilege/healthcheck` endpoint as an example.

Reply:
Cool. I don't think we want to have it be at that endpoint though. It should be at a different resource or a different port all together, since we don't want API calls from apigee to be able to access the healthcheck endpoint.